### PR TITLE
Add reference id to MTO payload

### PR DESCRIPTION
**Pull request by rswerve**
## Description

The move transfer order should now include a reference number, which we're calling the reference_id.  This PR adds that number to the MTO payload.

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

The Reference ID should be present on the move task order.

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/169582181) for this change

### CODE REVIEW

File: pkg/services/move_task_order/move_task_order.go [hunk 1]
```diff
@@ -99,6 +99,9 @@ func (f fetchMoveTaskOrder) UpdateMoveTaskOrderStatus(moveTaskOrderID uuid.UUID,
 	if err != nil {
 		return &models.MoveTaskOrder{}, err
 	}
+	if mto.Status != status && status == models.MoveTaskOrderStatusDraft {
+		mto.ReferenceID, _ = models.GenerateReferenceID(f.db)
```
Comment (mkrump) [inline pkg/services/move_task_order/move_task_order.go]: should be error checking this. i think that i may have put this ignore in here, we were mapping things out.

**Comment by mkrump** [comment]
Another thing (in line with Chris's comment), is it might be good to have a test in`pkg/services/move_task_order/move_task_order_test.go` alongside `TestMoveTaskOrderStatusUpdater`, showing when status is updated to `DRAFT` the reference id is correctly associated w/ the MTO.

File: pkg/services/move_task_order/move_task_order.go (hunk 1 above)
Comment (rswerve) [reply pkg/services/move_task_order/move_task_order.go]: Nope, that was me fixing a compiler error.

**Comment by rswerve** [comment]
Closing this, re-opening with new branch without weird git issues.

**Pull request closed without merging**